app/views/sessions/new.erb

```diff
@@ -0,0 +1,9 @@
+<h2>Please sign in.</h2>
+
+<form method="post" action="/sessions">
+  <h3>Email:</h3> 
```

Comment:
`<h3>` is not the correct semantic tag here, you want to use `<label>`

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form